src/main/java/org/opendevstack/provision/services/JiraAdapter.java

```diff
@@ -453,6 +443,30 @@ public OpenProjectData createBugtrackerProjectForODSProject(OpenProjectData proj
     }
   }
 
+  public void addWebhookProxyUrlToJiraProject(
+      String projectKey, String projectType, String webhookSecret) throws IOException {
+
+    String addProjectProperty =
+        jiraProjectTypePropertyCalculator.calculateProperty(
+            projectType, ADD_WEBHOOK_PROXY_URL_AS_PROJECT_PROPERTY, Boolean.FALSE.toString());
+
+    if (!Boolean.valueOf(addProjectProperty)) {
+      logger.debug(
+          "Project type is not configured for the setup of webhook proxy url as jira project property [projectKey={}, projectType={}]",
+          projectKey,
+          projectType);
+      return;
+    }
+
+    logger.info(
+        "Project type is not configured for the setup of webhook proxy url as jira project property [projectKey={}, projectType={}]",
```

Comment:
Isn't this what was checked in 453?

Reply:
good spotted... actually the intention here is log a message but the message is wrong... I'll fix it

Reply:
remove the word `not` and pushed the change